build/test.gulp.js

```diff
@@ -100,6 +108,14 @@ module.exports = function init(gulp = require('gulp')) {
 	 * ```bash
 	 * npx gulp test:component:run --name b-button --browsers ff,chrome
 	 * ```
+	 *
+	 * If you prefer to create components at runtime – run with `dummy` component,
```

Comment:
If you prefer to create components at runtime, you can use a dummy component. Make sure that that all components you want to test are declared as dependencies into `index.js` file of the demo page.